src/main/steps/home/get.ts

```diff
@@ -1,14 +1,18 @@
 import { Response } from 'express';
 
 import { AppRequest } from '../../app/controller/AppRequest';
-import { getNextIncompleteStepUrl } from '../../steps';
+import { CHECK_ANSWERS_URL, YOUR_DETAILS_URL } from '../../steps/urls';
 
 export class HomeGetController {
   public get(req: AppRequest, res: Response): void {
     if (req.session.userCase.divorceOrDissolution !== res.locals.serviceType) {
       throw new Error('Invalid case type');
     }
 
-    res.redirect(getNextIncompleteStepUrl(req));
+    const isCasePartiallyComplete = Object.entries(req.session.userCase).some(
+      ([key, value]) => !['id', 'divorceOrDissolution'].includes(key) && value
```

Comment:
would id not always exist as either the user creates a case or retrieves their previous one, they will have an id, so this will always be false? also i assume there would always be a value right?

Reply:
This is checking if the key does not include `id` or `divorceOrDissolution`, but does have other values then it will redirect to the check your answers so far page.